packages/@ourworldindata/grapher/src/mapCharts/MapTooltip.tsx

```diff
@@ -215,95 +208,86 @@ export class MapTooltip extends React.Component<MapTooltipProps> {
             lineColorScale?.getColor(datum?.value) ?? "#333"
         )
 
+        const yColumn = this.sparklineTable.get(this.mapColumnSlug),
+            { min, max } = this.sparklineManager?.yAxisConfig ?? {},
+            valueLabel = datum ? formatValue(datum.value) : undefined,
+            minLabel = formatValue(min || 0),
+            maxLabel = formatValue(
+                (last(customValueLabels) || max || yColumn.max) ?? 0
+            )
```

Comment:
Ah, this is also going wrong sadly, as surfaced by the screenshot I sent on Slack as part of the bug bash 🫠

In particular, `min` and `max` are only really suggestions - them being present just ensures that the actual axis covers _at least_ `range(min, max)`, but it will absolutely expand if there are data points `< min` or `> max`. So, yeah, this needs quite some reworking to not be blatantly wrong.